create_auction.php

```diff
@@ -1,13 +1,18 @@
 <?php include_once("header.php")?>
+<?php include_once("database.php"); ?>

```

Comment:
We can ideally put all the includes in a single PHP tag, but can safely ignore this suggestion since it makes no performance difference.